pkg/webhook/validatingwebhook/validate_k8simagepuller_request.go

```diff
@@ -63,12 +63,12 @@ func (v CheClusterRequestValidator) validate(body []byte) []byte {
 	}, requestingUser)
 
 	if err != nil {
```

Comment:
In line 57 you allow requests from `system:` accounts. So users still can create image pullers using Service Accounts? Is it expected?

Reply:
Thank you, I realized this is not necessary, I removed it in my latest commit

Reply:
But now it means that if the request is from a service account then it will be rejected (getting the User will fail with a 404 error) it means we won't be able to use ArgoCD or any other pipeline to create the puller or I'm missing something?